e2e/fixtures.ts

```
@@ -0,0 +1,76 @@
+import { UserSettings } from '@/utils/types'
+
+// Extend the test fixture to include extension context and config setup
+export const test = base.extend<{
+  context: BrowserContext
+  extensionId: string
+  optionsPage: Page
+  popupPage: Page
+  setupConfig: (config?: Partial<UserSettings>) => Promise<void>
+}>({
+      // eslint-disable-next-line no-empty-pattern
+      context: async ({}, use) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)